pkg/reconciler/build/build.go

```diff
@@ -164,12 +198,22 @@ func (c *Reconciler) reconcile(ctx context.Context, build *buildapi.Build) error
 		} else {
 			buildMetadata, err = c.buildMetadataFromBuildPod(pod)
 			if err != nil {
-				return errors.Wrap(err, "failed to get build metadata from build pod")
+				return fmt.Errorf("failed to get build metadata from build pod: %v", err)
 			}
 		}
+
+		var attestDigest string
+		if c.FeatureFlags.GenerateSlsaAttestation {
+			attestDigest, err = c.attestBuild(ctx, build, buildMetadata, pod)
+			if err != nil {
+				return fmt.Errorf("attesting build: %v", err)
```

Comment:
Could we add a "failed to" like our other error messages?

```suggestion
				return fmt.Errorf("failed to attest build: %v", err)
```